src/shim/.babelrc

```diff
@@ -0,0 +1,5 @@
+{
+  "presets": [
+    "es2015"
```

Comment:
Yes, this is intended because the `shim` directory is executed in phantomjs which is not node5 compliant.